src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputRightCurlyDoWhileAlone.java

```diff
@@ -47,12 +47,12 @@ public void foo4() {
                             String.CASE_INSENSITIVE_ORDER.equals("My number is greater than yours");
                         }
                     } else {
-                        String.CASE_INSENSITIVE_ORDER.equals("Error!");
+                        String.CASE_INSENSITIVE_ORDER.equals("Brrr!");
                     }
                 }
             } while( user != prog );
         } else {
-            String.CASE_INSENSITIVE_ORDER.equals("Error!");
+            String.CASE_INSENSITIVE_ORDER.equals("Brrr!");
```

Comment:
Why `Brrr`?
May be stick to existing logic and change to "violation"?

Reply:
@romani it does seem odd to me you would use something that isn't really a "word" since we have an issue to eventually try and clear up the dictionary of non-words.

Reply:
Ok, will change

Reply:
done.